tests/acceptance/test_med_safety_dsl.py

```diff
@@ -0,0 +1,289 @@
+            - patient_demographics.select_column(patient_demographics.date_of_birth)
+        ).convert_to_years()
+        cohort.assert_results(age=65)
+
+    with subtest("sex"):
+        cohort.sex = patient_demographics.select_column(patient_demographics.sex)
+        cohort.assert_results(sex="female")
+
+    # Indicators
+    # ==========
+
+    # GIB01
+    # -----
+    with subtest("oral_nsaid"):
+        cohort.oral_nsaid = (
+            clinical_events.filter(clinical_events.code.is_in(oral_nsaid_codelist))
+            .filter(clinical_events.date >= index_date_minus_3_months)
+            .filter(clinical_events.date <= index_date)
+            .exists_for_patient()
+        )
+        cohort.assert_results(oral_nsaid=True)
+
+    with subtest("ppi"):
+        cohort.ppi = (
+            clinical_events.filter(clinical_events.code.is_in(ppi_codelist))
+            .filter(clinical_events.date >= index_date_minus_3_months)
+            .filter(clinical_events.date <= index_date)
+            .exists_for_patient()
+        )
+        cohort.assert_results(ppi=True)
+
+    with subtest("indicator_gib_risk", missing_feature="Arithmetic on BoolSeries"):
+        at_gib_risk = cohort.oral_nsaid & (cohort.age >= 65)  # noqa: F841
+        not_at_gib_risk = at_gib_risk & ~cohort.ppi  # noqa: F841
+
+        # Why does the following numerator/denominator pair need to be commented out (to
+        # avoid raising a TypeError) but the other two numerator/denominator pairs do
+        # not?
+        # cohort.indicator_gib_risk_numerator = not_at_gib_risk
+        # cohort.indicator_gib_risk_denominator = at_gib_risk
+
+        cohort.assert_results(indicator_gib_risk_numerator=None)
+        cohort.assert_results(indicator_gib_risk_denominator=None)
+
+    with subtest(
+        "indicator_gib_num_admissions",
+        missing_feature="Missing TPPBackend.hospital_admissions",
+    ):
+        # TPPBackend.hospitalizations exists but doesn't expose an admission_method
+        # property. It's also not clear whether hospitalizations.code ==
+        # hospital_admissions.primary_diagnosis.
+        cohort.indicator_gib_num_admissions = (
+            hospital_admissions.filter(
+                hospital_admissions.primary_diagnosis.is_in(gib_admissions_codelist)
+            )
+            .filter(
+                # TODO: Query an IntColumn with is_in
+                hospital_admissions.admission_method.is_in(
+                    EMERGENCY_ADMISSIONS_CODELIST
+                )
+            )
+            .filter(hospital_admissions.episode_is_finished is True)
+            .filter(hospital_admissions.admission_date >= index_date_minus_3_months)
+            .filter(hospital_admissions.admission_date <= index_date)
+            .count_for_patient()
+        )
+        cohort.assert_results(indicator_gib_num_admissions=None)
+
+    with subtest(
+        "indicator_gib_admission_numerator", missing_feature="Missing dependencies"
```

Comment:
I think this is a bit confusing. Maybe define this variable in the same bock as the dependencies? The order doesn't matter too much, does it?

Reply:
The two dependencies are in separate `subtest` blocks, because each has missing features. I will merge this block into the preceding block, so as not to misuse `subtest` blocks. See bdb21f1.